modules/openapi-generator/src/main/resources/swift5/api.mustache

```diff
@@ -138,7 +139,10 @@ extension {{projectName}}API {
                 }
                 observer.onCompleted()
             }
-            return Disposables.create()
+            
+            return Disposables.create{{^useURLSession}}(){{/useURLSession}}{{#useURLSession}} {
```

Comment:
@DavidHorvath I think here we can remove the checks on the URLSession, because now is alamofire also supports cancellation

Reply:
Oh I was neglectful... Sorry about this and thanks for your comments!

Reply:
fixed